runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangeLinks.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020, Shingyx <https://github.com/Shingyx>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.grandexchange;
+
+import net.runelite.client.util.LinkBrowser;
+
+class GrandExchangeLinks
+{
+	static void openGeLink(String name, int itemId)
+	{
+		final String url = "http://services.runescape.com/m=itemdb_oldschool/"
```

Comment:
I believe they recently changed this to always redirect to https, so it may be worth updating it here.

Reply:
I also made it use + instead of _ to avoid the redirect.

I.e. compare
https://services.runescape.com/m=itemdb_oldschool/Limpwurt+root/viewitem?obj=225
and
https://services.runescape.com/m=itemdb_oldschool/Limpwurt_root/viewitem?obj=225

The second redirects to https://secure.runescape.com/m=itemdb_oldschool/a=13/Limpwurt+root/viewitem?obj=225